src/main/resources/static/public/data/morris-data.js

```diff
@@ -0,0 +1,117 @@
+$(function() {
```

Comment:
waarom hebben we bootstrap & theme niet gewoon minified? al die files toevoegen is echt onzin